src/ng-select/lib/ng-select.component.ts

```diff
@@ -874,6 +877,12 @@ export class NgSelectComponent implements OnDestroy, OnChanges, OnInit, AfterVie
 		}
 	}
 
+	private _setFinalTabFocus() {
```

Comment:
Same as above

Reply:
Re-using in mergeGlobalConfig and in ngOnChanges